### ISSUE TITLE
Terminal is unsable with Screen Reader (NVDA)

### ISSUE THREAD

**Issue by chinwobble** [microsoft/terminal#4341]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.18363.592]
Windows Terminal version (if applicable): 0.8.10091.0

Any other software?
NVDA: 2019.2.1
```

# Steps to reproduce
1. NVDA
2. Open Microsoft Terminal
3. type something in the console like in any terminal like `dir`
4. None of the output is being reported 

# Expected behavior
The console output should be visible to screen reader users. 
You should be able to use keys to read out console output line by line.
Pressing shift and arrow keys should highlight text. The screen reader can then read out this text.

# Actual behavior
The console is completely unusable for screen reader users :(
Further, VIM is completely unusable too in this mode

**Comment by zadjii-msft** [comment]
I'm pretty sure the root cause of this is actually #2447, but @carlos-zamora can double-check me on that.

**Comment by DHowett-MSFT** [comment]
This is a /dup of #2447. Thanks!
